modules/i3s/src/types.ts

```diff
@@ -1,3 +1,272 @@
+import type {GLTFMaterial} from '@loaders.gl/gltf';
+import {Quaternion, Vector3} from '@math.gl/core';
+import {Mesh} from '@loaders.gl/gltf';
```

Comment:
`import type`, and add the dependencies per the eslint-warnings below

Reply:
Done. I don't have any eslint warnings below.